qt/python/mantidqt/widgets/plotconfigdialog/curvestabwidget/markertabwidget/view.py

```diff
@@ -69,3 +102,13 @@ def update_fields(self, curve_props):
 
     def set_apply_to_all_enabled(self, enable):
         self.apply_to_all_button.setEnabled(enable)
+
+    def set_colour_fields_enabled(self, style):
+        if 'face' not in MARKER_STYLES[style]:
+            self.face_color_selector_widget.setDisabled(True)
+        else:
+            self.face_color_selector_widget.setDisabled(False)
+        if 'edge' not in MARKER_STYLES[style]:
+            self.edge_color_selector_widget.setDisabled(True)
+        else:
+            self.edge_color_selector_widget.setDisabled(False)
```

Comment:
These could probably be shortened to `self.face_color_selector_widget.setDisabled("face" not in MARKER_STYLES[style])`